lib/game/addDeveloperProduct.js

```diff
@@ -28,36 +27,26 @@ const nextFunction = (jar, token, universeId, name, priceInRobux, description) =
   }).then((res) => {
     if (res.Success && res.Message === 'Name available') {
       return http({
-        url: '//www.roblox.com/places/developerproducts/add',
+        url: `//develop.roblox.com/v1/universes/${universeId}/developerproducts?name=${name}&description=${description}&priceInRobux=${priceInRobux}`,
         options: {
           method: 'POST',
           jar: jar,
           headers: {
             'X-CSRF-TOKEN': token
           },
-          form: {
-            universeId,
-            name,
-            priceInRobux,
-            description
-          },
           resolveWithFullResponse: true
         }
       }).then((res) => {
         if (res.statusCode === 200) {
-          const $ = parser.load(res.body)
-          const creationStatus = $('#DeveloperProductStatus')
-
-          if (creationStatus.length > 0) {
-            return {
-              universeId,
-              name,
-              priceInRobux,
-              description,
-              productId: Number(creationStatus.text().match(/\d+/)[0])
-            }
+          return {
+            universeId,
+            name,
+            priceInRobux,
+            description,
+            productId: typeof res.body === 'object' ? res.body.id : JSON.parse(res.body).id
           }
         } else {
+          console.error(res)
```

Comment:
Is there a better way we can output this error? Could it be included as part of the throw?
While the method currently provides a pretty unhelpful error message this could be a good opportunity to fix this and throw the actual error.

Reply:
Line 50, check last commit to this pull.
Feel free to name changes to the way it throws the wire
error. I suppose adding status codes.

Reply:
Or did I forget to remove it again.

Reply:
[View](https://github.com/noblox/noblox.js/pull/639/commits/0653f49fe3d09c82bfb416a3bb460979ec1261a7)